node/lib/package/PackageInfo.ts

```diff
@@ -181,3 +184,25 @@ export class PackageTranslatorInfo {
      */
     public readonly onboardingProperties: { [propertyName: string]: string };
 }
+
+export class PackageOnboardingInfo
+{
+    public readonly onboardingId: string;
```

Comment:
When loading the package info, I think this should be resolved to a module name. Look at where that's done for schema and translator module names.